chia/types/spend_bundle.py

```diff
@@ -43,10 +44,14 @@ def aggregate(cls, spend_bundles) -> "SpendBundle":
         return cls(coin_spends, aggregated_signature)
 
     # TODO: this should be removed
```

Comment:
Mostly curious why this has a `this should be removed` todo?

Reply:
the wallet calls this function as if it's cheap. It's frequently called unnecessarily as well. I envision this being a free function whose name makes it clear that it's not cheap.